ios/brave-ios/Tests/BraveSharedTests/URLExtensionTests.swift

```diff
@@ -256,6 +256,31 @@ class URLExtensionTests: XCTestCase {
     XCTAssertFalse(URL.isValidURLWithoutEncoding(text: "https://google.ca/search?q=hello world"))
     XCTAssertFalse(URL.isValidURLWithoutEncoding(text: "hello world"))
   }
+
+  // swift-format-ignore: lineLength
+  func testETLDPlusOne() {
+    XCTAssertEqual((URL(string: "http://brave.com") as? NSURL)?.etldPlusOne, "brave.com")
```

Comment:
we should add `baseDomain` tests that will assert that the base domain of `something.github.io` is `something.github.io` and not `github.io`